pipelines/ocw/ocw-to-hugo.yml

```diff
@@ -22,9 +46,14 @@ jobs:
       run:
         args:
         - -exc
-        - |
+        - >
```

Comment:
This will break unless we add an && between the `yarn` and `node` commands because the `>` operator will render all following contents onto one line, whereas the `|` operator will retain newlines.

Reply:
I ended up putting the && after the `yarn` step.  The yarn lint pre-commit hook thinks the line is too long and keeps moving the `--strips3` argument to its own line which would run `ocw-to-hugo` without it.